.github/workflows/on_push_pr_fake_prerelease.yaml

```diff
@@ -0,0 +1,66 @@
+name: Push/PR fake prereleases
+
+on:
+  workflow_call:
+    inputs:
+      run_static_analysis:
+        required: false
+        type: boolean
+        default: true
+      run_test_nix:
+        required: false
+        type: boolean
+        default: true
+      run_test_windows:
+        required: false
+        type: boolean
+        default: true
+      run_integration_nix:
+        required: false
+        type: boolean
+        default: true
+      run_test_build_nix:
+        required: false
+        type: boolean
+        default: true
+      run_test_build_windows:
+        required: false
+        type: boolean
+        default: true
+
+jobs:
+  static-analysis:
+    if: ${{ inputs.run_static_analysis }}
+    uses: ./.github/workflows/static_analysis.yaml
+    with:
+      run_job: ${{ inputs.run_static_analysis }}
+
+  test-nix:
+    if: ${{ inputs.run_test_nix }}
+    uses: ./.github/workflows/test_nix.yaml
+    with:
+      run_job: ${{ inputs.run_test_nix }}
+
+  test-windows:
+    if: ${{ inputs.run_test_windows }}
+    uses: ./.github/workflows/test_windows.yaml
+    with:
+      run_job: ${{ inputs.run_test_windows }}
+
+  test-integration-nix:
+    if: ${{ inputs.run_integration_nix }}
+    uses: ./.github/workflows/test_integration_nix.yaml
+    with:
+      run_job: ${{ inputs.run_integration_nix }}
+
+  test-build-nix:
+    if: ${{ inputs.run_test_build_nix }}
+    uses: ./.github/workflows/test_build_nix.yaml
+    with:
+      run_job: ${{ inputs.run_test_build_nix }}
```

Comment:
can't we use the other job below instead?

Reply:
I rewrite them, if you could review it again please 🙏